geps/gep-1911.md

```diff
@@ -38,6 +38,8 @@ A Gateway implementation MUST recognize the Kubernetes Standard Application Prot
 
 Thus when a `*Route` points to a Kubernetes Service resource the backend protocol for each port can be specified by
 
+- setting the `protocol` field on the Kubernetes `Service`
+- setting the `protocol` field on the `Endpoint`/`EndpointSlice` object associated with a Kubernetes `Service`
 - setting the `appProtocol` field on the Kubernetes `Service`
 - setting the `appProtocol` field on an `Endpoint`/`EndpointSlice` object associated with a Kubernetes `Service`
```

Comment:
Sorry I missed this the first time around, but I think we should just focus on the values that are set on a Service (longer comment above).